Neos.Neos/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -107,6 +107,10 @@ additionalProperties:
 
                       'placeholder': { type: string, description: "Optional text that will be rendered in the aloha editor when there is no content." }
 
+                      'sampleCase' : { type: any }
+
+                      'fallbackCase' :  { type: any }
```

Comment:
these two aren't actual properties supported by Aloha, they seem to be used in the test for some reason?